redash/models/__init__.py

```diff
@@ -187,9 +187,7 @@ def get_schema(self, refresh=False):
 
         if cache is None:
             query_runner = self.query_runner
-            schema = sorted(
-                query_runner.get_schema(get_stats=refresh), key=lambda t: t["name"]
-            )
+            schema = query_runner.get_schema(get_stats=refresh)
```

Comment:
Why not do all the sorting here and then store the sorted version in cache?

Reply:
I'm not picky. I put it in the Resource code because key sorting seems more like a "view concern" than a "model concern". But it really doesn't matter.

Reply:
I moved this logic all into the model.